### PR TITLE
Validation & Accuracy Fix

### PR DESCRIPTION
**Pull request by ZhangBanger**
DNM - do not merge

### CODE REVIEW

**Comment by ZhangBanger** [comment]
Ok to merge! 

```
/Users/andy/anaconda/bin/python /Users/andy/code/kaggle-iEEG-prediction/train.py
Step 0: train-loss = 3.90 (13.697 sec) | train-acc = 0.797 || valid-loss = 2.91 | valid-acc = 0.924
Step 1: train-loss = 5.19 (6.508 sec) | train-acc = 0.875 || valid-loss = 3.69 | valid-acc = 0.902
Step 2: train-loss = 5.10 (7.276 sec) | train-acc = 0.875 || valid-loss = 3.44 | valid-acc = 0.893
Step 3: train-loss = 2.59 (5.622 sec) | train-acc = 0.844 || valid-loss = 2.70 | valid-acc = 0.901
Step 4: train-loss = 1.58 (5.465 sec) | train-acc = 0.891 || valid-loss = 2.71 | valid-acc = 0.892
Step 5: train-loss = 3.04 (5.597 sec) | train-acc = 0.859 || valid-loss = 2.19 | valid-acc = 0.911
Step 6: train-loss = 3.36 (6.363 sec) | train-acc = 0.812 || valid-loss = 2.35 | valid-acc = 0.909
Step 7: train-loss = 5.87 (6.269 sec) | train-acc = 0.812 || valid-loss = 2.11 | valid-acc = 0.912
```

**Comment by ZhangBanger** [comment]
Add precision/recall/f1 eval metrics

**Comment by ZhangBanger** [comment]
closes #7
